dev/TabView/TabView.cpp

```diff
@@ -231,16 +231,32 @@ void TabView::OnListViewLoaded(const winrt::IInspectable&, const winrt::RoutedEv
     if (auto listView = m_listView.get())
     {
         // Now that ListView exists, we can start using its Items collection.
-        auto items = TabItems();
-        auto numItemsToCopy = static_cast<int>(items.Size());
-        if (auto lvItems = listView.Items())
+        
+        if (!listView.ItemsSource())
         {
-            for (int i = 0; i < numItemsToCopy; i++)
+            if (auto const lvItems = listView.Items())
             {
-                // App put items in our Items collection; copy them over to ListView.Items
-                lvItems.Append(items.GetAt(i));
+                auto items = TabItems();
```

Comment:
Since this local is only ever used in the for loop declaration can we delete it and change the declaration to `for (auto const item : TabItems())`